wdl/EvidenceQC.wdl

```diff
@@ -169,5 +196,83 @@ workflow EvidenceQC {
     File bincov_matrix = MakeBincovMatrix.merged_bincov
     File bincov_matrix_index = MakeBincovMatrix.merged_bincov_idx
     File bincov_median = MedianCov.medianCov
+
+    File? qc_table_csv = MakeQcTable.qc_table_csv
+  }
+}
+
+task MakeQcTable {
+  input {
+
+    File ploidy_plots
+    File bincov_median
+    File WGD_scores
+    Array[Float]? melt_insert_size
+    Array[String] samples
+
+
+    File? manta_qc_low
+    File? manta_qc_high
+    File? melt_qc_low
+    File? melt_qc_high
+    File? wham_qc_low
+    File? wham_qc_high
+    File? scramble_qc_low
+    File? scramble_qc_high
+
+    String sv_pipeline_docker
+    String output_prefix
+    RuntimeAttr? runtime_override
   }
+
+  output {
+    File qc_table_csv = "${output_prefix}.evidence_qc_table.tsv"
+  }
+
+  command <<<
+    set -euxo pipefail
```

Comment:
The -x option is super useful for debugging but generally I prefer to remove it once development is complete because it makes the log files overly large
```suggestion
    set -euo pipefail
```